templates/test/previous.html.ep

```diff
@@ -1,5 +1,5 @@
 <div id="scenario">
-    <div class="h5">Results for <i><%= $scenario %></i></div>
+    <div class="h5">Results for <i><%= link_to($scenario => url_for ('latest')->query($job->scenario_hash)) %></i></div>
```

Comment:
latest results implies it's multiple results, but it's just one -> 'latest job' then